sdk/ml/azure-ai-ml/azure/ai/ml/operations/_workspace_operations.py

```diff
@@ -91,18 +95,25 @@ def list(self, *, scope: str = Scope.RESOURCE_GROUP) -> Iterable[Workspace]:
                 :caption: List the workspaces by resource group or subscription.
         """
 
+        # Kind should be converted to a comma-separating string if multiple values are supplied.
+        formatted_kinds = filtered_kinds
+        if isinstance(filtered_kinds, list):
+            formatted_kinds = ",".join(filtered_kinds)  # type: ignore[arg-type]
```

Comment:
Consider flipping the comparison from "is a list" to "is not a string", so that users can specify any `Iterable[str]` and not just `List[str]`

```suggestion
        formatted_kinds = filtered_kinds
        if not isinstance(filtered_kinds, str):
            formatted_kinds = ",".join(filtered_kinds)  # type: ignore[arg-type]
```